recipe/esmf-8-4-regrid.patch

```diff
@@ -0,0 +1,19 @@
+diff --git a/ni/src/examples/gsun/ESMF_regridding.ncl b/ni/src/examples/gsun/ESMF_regridding.ncl
+index 2db5e1e93..750e17c62 100644
+--- a/ni/src/examples/gsun/ESMF_regridding.ncl
++++ b/ni/src/examples/gsun/ESMF_regridding.ncl
+@@ -2897,12 +2897,12 @@ begin
+ 
+ ;---Checking if the user specifies that source grid is ESMF
+     if (isatt_logical_true(opt2,"SrcESMF")) then
+-        ESMFCMD = ESMFCMD + " --src_type ESMF"
++        ESMFCMD = ESMFCMD + " --src_type ESMFMESH"
```

Comment:
The ``--src_type`` and ``--dst_type`` flags were ignored in older ESMF versions and reinstated with different values in (compare [v8.3](https://earthsystemmodeling.org/docs/release/ESMF_8_3_0/ESMF_refdoc/node3.html) vs [v8.4](https://earthsystemmodeling.org/docs/release/ESMF_8_4_0/ESMF_refdoc/node3.html)).